kubernetes/check.py

```diff
@@ -547,6 +554,12 @@ def _update_kube_events(self, instance, pods_list, event_items):
             title = '{} {} on {}'.format(involved_obj.get('name'), event.get('reason'), node_name)
             message = event.get('message')
             source = event.get('source')
+            k8s_event_type = event.get('type')
+            if k8s_event_type in K8S_ALERT_MAP:
```

Comment:
`alert_type = K8S_ALERT_MAP.get(k8s_event_type, 'info')`?

Reply:
of course, that's much better :-)